src/pages/Quiz/QuizPage.jsx

```diff
@@ -0,0 +1,120 @@
+import { useState } from "react";
+import PageLayout from "../../components/PageLayout";
+import Button from "@mui/material/Button";
+import LinearProgress from '@mui/material/LinearProgress';
+import { v4 as uuidv4 } from "uuid";
+import { Box } from "@mui/system";
+
+/**
+ * Renders the Assessment page.
+ * @returns {JSX.Element} QuizPage component
+ */
+
+function QuizPage() {
+
+    const [currentQuestion, setcurrentQuestion] = useState(0);
+    const [progressValue, setprogressValue] = useState(1);
+
+    const handleAnswerButtonClick = (answer) => {
+        if (quiztions[currentQuestion].correct === answer.target.textContent) // checks if the answer is correct
+            console.log("Correct");
+        else
+            console.log("Incorrect");
+
+        if (currentQuestion !== quiztions.length - 1) { // updates questions and progress bar
+            setcurrentQuestion(currentQuestion + 1);
+            setprogressValue((((currentQuestion +1)/ quiztions.length) * 100));
+        }
+
+        // update the progress bar on the last question
+        if (currentQuestion === quiztions.length-1)
+            setprogressValue(100); 
+    }
+
+    const quiztions = [ // placeholder for the real questions
+        {
+          question: "Question 1",
+          answers: ["Answer 1", "Answer 2", "Answer 3"],
+          correct: "Answer 1",
+        },
+        {
+            question: "Question 2",
+            answers: ["Answer 1", "Answer 2", "Answer 3", "Answer 4"],
+            correct: "Answer 2",
+        },
+        {
+            question: "Question 3",
+            answers: ["Answer 1", "Answer 2", "Answer 3"],
+            correct: "Answer 3",
+        },
+      ];
+      
+
+    return (
+        <PageLayout title="Quiz" viewportPage>
+    
+            <Box
+                sx={{
+                flexGrow: 1,
+                display: "flex",
+                flexDirection: "column",
+                justifyContent: "center",
+                alignItems: "center",
+                height: "calc(100% - 60px)",
+                }}
+            >
+                <LinearProgress 
+                    sx={{
+                    margin: "4% 0 3% 0",
+                    height: "1.5%", 
+                    width: "90%"
+                    }} 
+                    variant="determinate" 
+                    value={progressValue} 
+                /> 
+
+                <Box
+                    sx={{
+                    height: "80%",
+                    width: "80%",
+                    backgroundColor: "#D9D9D9",
```

Comment:
Please you theme data for colors

Reply:
I don't quite understand. The grey tone is the same as in the Refined Wireframe and the assessment page. Should I import the theme from somewhere?

Reply:
The assessment page is right now only a placeholder. So, it is not really part of the theme. The Background color of the Quiz will be final and should reflect the theme. So, please you theme data

Reply:
The background color is just the same as in the wire frame and I thought that was fine. 
As I still don't quite understand, would you mind rephrasing what I need to do?
Is the grey box also supposed to turn Blue like the buttons?

Reply:
![Screenshot 2023-11-13 at 19 51 26](https://github.com/SE2-WS23/Skillverse-Web/assets/136974808/5259284f-1c61-4ef8-a4e9-1c04447a7c3a)
:(

I have yet to push this and would like clarification whether this is what you meant.

Reply:
![Screenshot 2023-11-15 at 15 25 31](https://github.com/SE2-WS23/Skillverse-Web/assets/136974808/9d512e0d-1e5c-4093-a75c-edf0c3649621)


Quiz with the new light blue, please say if this is ok